code/mob/living/carbon/human/monkeys.dm

```diff
@@ -260,10 +260,10 @@
 					var/datum/bioEffect/power/healing_touch/healing_touch = src.bioHolder.GetEffect("healing_touch")
 					var/datum/bioEffect/power/midas/midas_touch = src.bioHolder.GetEffect("midas")
 					var/datum/bioEffect/power/ink/ink_glands = src.bioHolder.GetEffect("inkglands")
-					if (ismob(thing_to_poke) && healing_touch?.ability.last_cast < world.time)
-						healing_touch.ability.handleCast(thing_to_poke)
-					else if (!ismob(thing_to_poke) && midas_touch?.ability.last_cast < world.time)
-						midas_touch.ability.handleCast(thing_to_poke)
+					if (ismob(thing_to_poke) && healing_touch && healing_touch?.ability.last_cast < world.time)
```

Comment:
```suggestion
					if (ismob(thing_to_poke) && healing_touch && healing_touch.ability.last_cast < world.time)
```
? is redundant since you check just before

Reply:
i just wanna be really sure ok :(